sdk/src/main/java/it/ministerodellasalute/verificaC19sdk/data/VerifierRepositoryImpl.kt

```diff
@@ -326,9 +322,11 @@ class VerifierRepositoryImpl @Inject constructor(
     }
 
     private fun checkCurrentDownloadSize() {
-        val config =
-            RealmConfiguration.Builder().name(REALM_NAME).allowQueriesOnUiThread(true)
-                .build()
+        val config = RealmConfiguration.Builder()
```

Comment:
Don't want to bother but this RealmConfiguration.Builder block should be extracted and moved to some unique point for not to repeat code and, in case of migration, you should not search and replate for new schema version and such.

Reply:
Anyway out of this PR's scope (_"keep it simple" fix_). If/when there would be a required migration owing to any RevokedPass object schema changes, the code would be obviously re-arranged.